apps/modernization-ui/src/pages/patient/profile/phoneEmail/PhoneEmailEntryForm.tsx

```diff
@@ -97,18 +98,30 @@ export const PhoneEmailEntryForm = ({ action, entry, onChange, onCancel }: Entry
                             control={control}
                             name="countryCode"
                             defaultValue={entry.countryCode}
-                            render={({ field: { onChange, value } }) => (
-                                <Input
-                                    flexBox
-                                    onChange={onChange}
-                                    defaultValue={value}
-                                    type="text"
-                                    label="Country code"
-                                    name="countryCode"
-                                    htmlFor="countryCode"
-                                    id="countryCode"
-                                />
-                            )}
+                            rules={{
+                                pattern: {
+                                    value: /^\+?\d{1,6}$/,
+                                    message: 'Invalid country code'
```

Comment:
Two things here, when using the rules the `message` with be available in `error.message` so for the Input property you can use `error={error?.message}`.

For the message let's try to be proactive for 508 compliance,  can the validation message be changed to "A country code should be 1 to 3 digits"

Reply:
sounds good, thanks